Stratis.Bitcoin/Features/Wallet/IWalletManager.cs

```diff
@@ -37,11 +37,10 @@ public interface IWalletManager : IDisposable
 
         /// <summary>
         /// Loads a wallet from a file.
-        /// </summary>
-        /// <param name="password">The user's password.</param>
+        /// </summary>        
         /// <param name="name">The name of the wallet.</param>
         /// <returns>The wallet.</returns>
-        Wallet LoadWallet(string password, string name);
+        Wallet LoadWallet(string name);
```

Comment:
Ok, I'll put it back in. I think there was another place where I removed a parameter I have to check.